exercises.ipynb

```diff
@@ -150,22 +171,104 @@
     "# HINT HINT HINT: There's no reason you shouldn't be able to reuse the previous \n",
     "# functions (get_titles and parse_bricks)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 102,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "https://brickset.com/sets/year-2016\n",
+      "https://brickset.com/sets/year-2016/page-2\n",
+      "https://brickset.com/sets/year-2016/page-3\n",
+      "https://brickset.com/sets/year-2016/page-4\n",
+      "https://brickset.com/sets/year-2016/page-5\n",
+      "https://brickset.com/sets/year-2016/page-6\n",
+      "https://brickset.com/sets/year-2016/page-7\n",
+      "https://brickset.com/sets/year-2016/page-8\n",
+      "https://brickset.com/sets/year-2016/page-9\n",
+      "https://brickset.com/sets/year-2016/page-10\n",
+      "https://brickset.com/sets/year-2016/page-11\n",
+      "https://brickset.com/sets/year-2016/page-12\n",
+      "https://brickset.com/sets/year-2016/page-13\n",
+      "https://brickset.com/sets/year-2016/page-14\n",
+      "https://brickset.com/sets/year-2016/page-15\n",
+      "https://brickset.com/sets/year-2016/page-16\n",
+      "https://brickset.com/sets/year-2016/page-17\n",
+      "https://brickset.com/sets/year-2016/page-18\n",
+      "https://brickset.com/sets/year-2016/page-19\n",
+      "https://brickset.com/sets/year-2016/page-20\n",
+      "https://brickset.com/sets/year-2016/page-21\n",
+      "https://brickset.com/sets/year-2016/page-22\n",
+      "https://brickset.com/sets/year-2016/page-23\n",
+      "https://brickset.com/sets/year-2016/page-24\n",
+      "https://brickset.com/sets/year-2016/page-25\n",
+      "https://brickset.com/sets/year-2016/page-26\n",
+      "https://brickset.com/sets/year-2016/page-27\n",
+      "https://brickset.com/sets/year-2016/page-28\n",
+      "https://brickset.com/sets/year-2016/page-29\n",
+      "https://brickset.com/sets/year-2016/page-30\n",
+      "https://brickset.com/sets/year-2016/page-31\n",
+      "https://brickset.com/sets/year-2016/page-32\n",
+      "https://brickset.com/sets/year-2016/page-33\n",
+      "https://brickset.com/sets/year-2016/page-34\n"
+     ]
+    }
+   ],
+   "source": [
+    "# note: reusing the previous functions isn't ideal because parse_bricks loads the page but discards the loaded page\n",
+    "# so we'll just reuse get_titles and not parse_bricks\n",
+    "\n",
+    "def get_next(soup):\n",
```

Comment:
This is fine, but using CSS notation would avoid the nested if statements! Also, you could avoid the nested if statements by performing a TRY/EXCEPT!